Source/Scene/MetadataClassProperty.js

```diff
@@ -290,8 +272,6 @@ Object.defineProperties(MetadataClassProperty.prototype, {
  *
  * @param {*} value The integer value or array of integer values.
  * @returns {*} The normalized value or array of normalized values.
- *
- * @private
  */
 MetadataClassProperty.prototype.normalize = function (value) {
```

Comment:
I think we should keep `normalize` and `unnormalize` private for now until we need to expose them publicly.